app/routes/__frontend/newsletter/issues/2022-10.jsx

```diff
@@ -0,0 +1,319 @@
+export const handle = {
+	meta: {
+		title: 'Virtual Coffee Newsletter, October 2022',
+		description: 'Hacktoberfest is here! 💝',
+	},
+	date: '2022-10-01',
+	listTitle: 'October 2022',
+};
+
+export const meta = () => {
+	return handle.meta;
+};
+
+export default function Issue() {
+	return (
+		<>
+			<h2>Hey friends!</h2>
+
+			<p className="lead">
+				It's time to get our open source on! Hacktoberfest is HERE.
+			</p>
+
+			<hr />
+
+			<h2>💞 Community Kindness</h2>
+			<p>
+				<em>Spotlighting some of the kindness happening in our community.</em>
+			</p>
+			<blockquote className="blockquote">
+				<p className="mb-0">
+					"Just wanted to say how much I love the folks here. All the good
+					things in my life are better because of you all, either directly or
+					indirectly. And this is where I've made the most friends in a long
+					time."
+				</p>
+				<footer className="blockquote-footer">Kirk</footer>
+			</blockquote>
+			<blockquote className="blockquote">
+				<p className="mb-0">
+					"I love and appreciate you all and thank you for helping me as a
+					person."
+				</p>
+				<footer className="blockquote-footer">Nerando</footer>
+			</blockquote>
+
+			<hr />
+
+			<h2 className="my-5">📆 What's happening at Virtual Coffee</h2>
+
+			<h3 className="font-italic">September Recap</h3>
+			<p>
+				Monthly challenge -&gt;{' '}
+				<a href="https://virtualcoffee.io/monthlychallenges/sept-2022">
```

Comment:
Can we try to use `Link` for internal links?

```suggestion
				<Link to="/monthlychallenges/sept-2022">
```